command/detect.go

```diff
@@ -40,9 +39,9 @@ func DetectCommand(detect libjavabuildpack.Detect, metadata riff_buildpack.Metad
 	if err != nil {
 		return false, err
 	}
-	if info.Mode().Perm() & 0100 == 0100 {
+	if info.Mode().Perm()&0100 == 0100 {
 		return true, nil
 	} else {
-		return false, fmt.Errorf("missing exec permission on %s", path)
+		return false, nil
```

Comment:
Maybe a debug log message here?  Since the lack of execute permissions is quite hidden from a user which will be screaming "BUT THE FILE IS THERE!" it's good to have some sort of extra information.

Reply:
Good idea